client/signup/steps/import/ready/index.tsx

```diff
@@ -145,8 +145,47 @@ const ReadyStep: React.FunctionComponent< PropsWithoutUrl > = ( { platform } ) =
 	);
 };
 
+const ReadyAlreadyOnWPCOM: React.FunctionComponent< Props > = ( { urlData } ) => {
+	const { __ } = useI18n();
+
+	return (
+		<div className="import-layout__center">
+			<div className="import__header">
+				<div className="import__heading  import__heading-center">
+					<Title>{ __( 'Your site is already on WordPress.com' ) }</Title>
+					<SubTitle>
+						{ createInterpolateElement(
+							sprintf(
+								/* translators: the website could be any domain (eg: "yourname.com") */
+								__(
+									'It looks like <strong>%(website)s</strong> is already on WordPress.com. Try a different address or start buidling a new site instead.'
+								),
+								{
+									website: convertToFriendlyWebsiteName( urlData.url ),
+								}
+							),
+							{ strong: createElement( 'strong' ) }
+						) }
+					</SubTitle>
+
+					<div className="import__buttons-group">
+						<NextButton>{ __( 'Start building' ) }</NextButton>
+						<div>
+							<BackButton>{ __( 'Back to start' ) }</BackButton>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Back to the start' )` *ES Score: 6*